python/cudf/cudf/tests/test_repr.py

```diff
@@ -13,7 +13,7 @@
 from cudf.testing import _utils as utils
 from cudf.utils.dtypes import np_dtypes_to_pandas_dtypes
 
-repr_categories = utils.NUMERIC_TYPES + ["str", "category", "datetime64[ns]"]
+repr_categories = ["int64", "float64", "str", "category", "datetime64[ns]"]
```

Comment:
Here I reason that we only need one of each `kind` because every flavor of float or int should have the same `__repr__`.

Reply:
> Here I reason that we only need one of each `kind` because every flavor of float or int should have the same `__repr__`.

Agree, can we also add `uint16` to cover unsigned int types?

Reply:
Also, if the tests can handle `bool` might try adding it here aswell. Else okay to skip it.

Reply:
I have added `uint`. It seems `bool` was not originally included here, and adding it now creates some failed tests that use this parameterization. We will probably need to update a bunch of them to cover `bool`. Want me to create an issue around this?

Reply:
I see, not needed for a new issue if we are covering bool atleast anywhere in our repr testing. Else, we probably want to cover it as part of another PR.